### PR TITLE
Correct the clusterId for monitoring

### PR DESCRIPTION
**Pull request by asalkeld**
```release-note
MSFT: ACS_RESOURCE_NAME is corrected to be AKS_RESOURCE_ID (the value is the full resourceId) and added AKS_REGION
```
part of https://github.com/openshift/openshift-azure/issues/1993

### CODE REVIEW

**Comment by asalkeld** [comment]
note @charlesakalugwu had to do this some time back: 07786a244d34d3b03cd4bee7166c54b5b8e94a90 it would be good to know why that didn't work.

File: pkg/sync/v10/translate.go [hunk 1]
```diff
@@ -332,8 +332,12 @@ var translations = map[string][]struct {
 			Template: "{{ .Config.Images.LogAnalyticsAgent }}",
 		},
 		{
-			Path:     jsonpath.MustCompile("$.spec.template.spec.containers[0].env[?(@.name='ACS_RESOURCE_NAME')].value"),
-			Template: "{{ .ContainerService.Name }}",
+			Path:     jsonpath.MustCompile("$.spec.template.spec.containers[0].env[?(@.name='AKS_RESOURCE_ID')].value"),
+			Template: "/subscriptions/{{ .ContainerService.Properties.AzProfile.SubscriptionID }}/resourceGroups/{{ .ContainerService.Properties.AzProfile.ResourceGroup }}/providers/Microsoft.ContainerService/openShiftManagedClusters/{{ .ContainerService.Name }}",
```
Comment (jim-minter) [inline pkg/sync/v10/translate.go]:
```suggestion
			Template: "{{ .ContainerService.ID }}",
```

File: pkg/sync/v10/translate.go [hunk 2]
```diff
@@ -374,8 +378,12 @@ var translations = map[string][]struct {
...
```
Comment (jim-minter) [inline pkg/sync/v10/translate.go]:
```suggestion
			Template: "{{ .ContainerService.ID }}",
```

**Comment by jim-minter** [comment]
> note @charlesakalugwu had to do this some time back: 07786a2 it would be good to know why that didn't work.

Today MSFT told me that they started with ACS_RESOURCE_NAME for PoC and confirmed that this change is what they want.

**Pull request merged**